shared/menubar/index.desktop.js

```diff
@@ -200,66 +198,69 @@ class MenubarRender extends React.Component<Props, State> {
     )
   }
 
-  _menuItems(countMap: Object, startingUp?: boolean = false) {
-    const wallet = Flags.walletsEnabled
-      ? [
-          {
-            onClick: () => this.props.openApp(Tabs.walletsTab),
-            title: 'Wallet',
-            view: this._menuView('Wallet', 'iconfont-nav-wallets', countMap[Tabs.walletsTab] || 0),
-          },
-        ]
-      : []
+  _menuItems() {
+    const countMap = this.props.badgeInfo || {}
+    const startingUp = this.props.daemonHandshakeState !== 'done'
+    const loggedOut = !this.props.username
 
-    const tabs = [
-      ...wallet,
-      {
-        onClick: () => this.props.openApp(Tabs.gitTab),
-        title: 'Git',
-        view: this._menuView('Git', 'iconfont-nav-git', countMap[Tabs.gitTab] || 0),
-      },
-      {
-        onClick: () => this.props.openApp(Tabs.devicesTab),
-        title: 'Devices',
-        view: this._menuView('Devices', 'iconfont-nav-devices', countMap[Tabs.devicesTab] || 0),
-      },
-      {
-        onClick: () => this.props.openApp(Tabs.settingsTab),
-        title: 'Settings',
-        view: this._menuView('Settings', 'iconfont-nav-settings', countMap[Tabs.settingsTab] || 0),
-      },
-    ]
+    const sectionTabs =
+      startingUp || loggedOut
+        ? []
+        : [
+            {
+              onClick: () => this.props.openApp(Tabs.walletsTab),
+              title: 'Wallet',
+              view: this._menuView('Wallet', 'iconfont-nav-wallets', countMap[Tabs.walletsTab]),
+            },
+            {
+              onClick: () => this.props.openApp(Tabs.gitTab),
+              title: 'Git',
+              view: this._menuView('Git', 'iconfont-nav-git', countMap[Tabs.gitTab]),
+            },
+            {
+              onClick: () => this.props.openApp(Tabs.devicesTab),
+              title: 'Devices',
+              view: this._menuView('Devices', 'iconfont-nav-devices', countMap[Tabs.devicesTab]),
+            },
+            {
+              onClick: () => this.props.openApp(Tabs.settingsTab),
+              title: 'Settings',
+              view: this._menuView('Settings', 'iconfont-nav-settings', countMap[Tabs.settingsTab]),
+            },
+            'Divider',
+          ]
+
+    const openApp = startingUp
+      ? []
+      : [{onClick: () => this.props.openApp(), title: 'Open main app'}, 'Divider']
 
-    const openMainApp = [{onClick: () => this.props.openApp(), title: 'Open main app'}]
+    const showFinder =
+      startingUp || loggedOut
+        ? []
+        : [
+            {onClick: () => this.props.showInFinder('/'), title: `Open folders in ${Styles.fileUIName}`},
```

Comment:
This literally just opens `/`, not `/` under KBFS. I'll fix it in my branch to open /keybase instead.